README.md

```diff
@@ -1,2 +1,11 @@
+my goals for this project
 
-![first plot](cloud/project/plots)
+1. show popular songs popularity over each decade
+
+2. Compare popular and unpopular songs in each different subclass, loudness, danceability, energy, instrumentalness, acousticness, valence, speechiness, tempo, key and liveness
+
+I found that song popularities do increase through out the 90s on into the 2000s I also found that theres no difference in each subclass so that means that unpopular songs try to copy or remix popular tracks
+
+![first plot](plots/plot1.png)
```

Comment:
After each plot, add your description of what you think that plot is telling us.